docs/source/userguide/command_line.rst

```diff
@@ -1,508 +1,520 @@
- | # # # This source code is protected under the license referenced at
- | # # # https://github.com/NRLMMD-GEOIPS.
+ This source code is protected under the license referenced at
+ https://github.com/NRLMMD-GEOIPS.
 
 .. _command_line:
 
-Command Line Interface
-**********************
+Command Line Interface (CLI)
+****************************
 
-.. _geoips:
+.. warning::
 
-:ref:`GeoIPS CLI <geoips>`
+    The CLI is currently under development.
+    The CLI may change without warning!
+    Please consult this documentation for up-to-date info on the CLI.
 
-The GeoIPS Command Line Interface (CLI) is currently under development, but we'll keep
-updated instructions in this documentation that reflect the current state of the CLI.
+The CLI interacts with GeoIPS artifacts.
+A GeoIPS artifact is a piece of data or functionality,
+such as an Interface, Package, Plugin,
+Family, Test Script, Test Dataset, etc.
 
-The GeoIPS CLI is used to interrogate / utilize GeoIPS ``artifacts`` in a user-friendly
-manner. A ``GeoIPS artifact`` is an term used to reflect the chosen piece of data or
-functionality implemented in GeoIPS. This could be a GeoIPS Interface, Package, Plugin,
-Family, Test Script, Test Dataset, etc. that is used and/or implemented throughout the
-GeoIPS Environment.
+CLI commands are split up into two groups by their utility:
 
-The CLI is split up into two groups as of right now. CLI commands which
-:ref:`retrieve information<information_retrieval>` from GeoIPS and CLI commands which
-:ref:`perform processes<performing_processes>` via GeoIPS. We'll dive into each of those
-sections now.
+ - `Information retrieval`_ commands
+ - `Action performing commands <#performing-processes>`_
 
-CLI Use Cases
-*************
+You can find the automatically created CLI usage documentation `here <./command_line_autodoc>`_.
 
-For your ease of use, we've provided a complete  listing of all available GeoIPS CLI
-commands below. This diagrams what commands are available, and the required and/or
-optional arguments that go alongside these commands.
-
-.. dropdown:: GeoIPS CLI Commands
-
-    .. admonition:: Usage: geoips
-
-        .. autoprogram:: geoips.commandline.commandline_interface:GeoipsCLI().parser
-            :prog: geoips
-
-.. _information_retrieval:
+.. contents:: Table of Contents
+    :local:
+    :backlinks: none
 
 Information Retrieval
 =====================
 
-The GeoIPS CLI Implements two top-level commands which retrieve information about GeoIPS
-artifacts based on the sub-commands provided. These two top-level commands are
-``describe`` (``desc``) and ``list`` (``ls``). Each command shown below can be ran with
-a ``-h`` flag to provide help instructions for the command you are running.
+The CLI Implements two top-level commands which retrieve information about GeoIPS
+artifacts: ``describe`` and ``list`` (``ls``).
 
-.. _geoips_describe:
+Each command shown below can be ran with a ``-h``
+flag to provide associated help instructions.
```

Comment:
Maybe move this to the top section and expand slightly? For example:

To retrieve information about the CLI's commands and arguments, the ``-h/--help`` argument can be provided at any time. For example, each of the following will return different, context dependent, help information:
- ``geoips -h``
- ``geoips list -h``
- ``geoips list algorithms -h``
- ``geoips run -h``